base/applications/regedit/lang/ro-RO.rc

```diff
@@ -83,57 +83,57 @@ BEGIN
         MENUITEM SEPARATOR
-        MENUITEM "Î&ncărcare registru…", ID_REGISTRY_LOADHIVE, GRAYED
-        MENUITEM "Eli&berare registru…", ID_REGISTRY_UNLOADHIVE, GRAYED
+        MENUITEM "Î&ncărcare de regiștri…", ID_REGISTRY_LOADHIVE, GRAYED
+        MENUITEM "&Descărcare de regiștri…", ID_REGISTRY_UNLOADHIVE, GRAYED
         MENUITEM SEPARATOR
         MENUITEM "&Conectare la un registru din rețea…", ID_REGISTRY_CONNECTNETWORKREGISTRY
-        MENUITEM "&Deconectare de la registru din rețea…", ID_REGISTRY_DISCONNECTNETWORKREGISTRY, GRAYED
+        MENUITEM "&Deconectare de la regiștrii din rețea…", ID_REGISTRY_DISCONNECTNETWORKREGISTRY, GRAYED
         MENUITEM SEPARATOR
         MENUITEM "I&mprimare…\tCtrl+P", ID_REGISTRY_PRINT, GRAYED
         MENUITEM SEPARATOR
         MENUITEM "I&eșire", ID_REGISTRY_EXIT
     END
     POPUP "&Editare"
     BEGIN
-        MENUITEM "&Modifică", ID_EDIT_MODIFY
+        MENUITEM "&Modificare", ID_EDIT_MODIFY
         MENUITEM SEPARATOR
-        POPUP "&Nouă"
+        POPUP "&Nou"
         BEGIN
             MENUITEM "&Cheie", ID_EDIT_NEW_KEY
             MENUITEM SEPARATOR
-            MENUITEM "&Valoare tip șir", ID_EDIT_NEW_STRINGVALUE
-            MENUITEM "Valoare 1 &bit", ID_EDIT_NEW_BINARYVALUE
-            MENUITEM "Valoare 32 biți (4 &octeți)", ID_EDIT_NEW_DWORDVALUE
-            MENUITEM "Valoare &multi-șir", ID_EDIT_NEW_MULTISTRINGVALUE
-            MENUITEM "Valoare șir e&xtensibil", ID_EDIT_NEW_EXPANDABLESTRINGVALUE
+            MENUITEM "&Valoare de tip șir", ID_EDIT_NEW_STRINGVALUE
+            MENUITEM "Valoare de 1 &bit", ID_EDIT_NEW_BINARYVALUE
+            MENUITEM "Valoare de 32 de biți (4 &octeți)", ID_EDIT_NEW_DWORDVALUE
+            MENUITEM "Valoare de șir &multiplu", ID_EDIT_NEW_MULTISTRINGVALUE
+            MENUITEM "Valoare de șir e&xtensibil", ID_EDIT_NEW_EXPANDABLESTRINGVALUE
         END
         MENUITEM SEPARATOR
         MENUITEM "&Permisiuni…", ID_EDIT_PERMISSIONS
         MENUITEM SEPARATOR
-        MENUITEM "Șt&erge\tDel", ID_EDIT_DELETE
-        MENUITEM "&Redenumește", ID_EDIT_RENAME
+        MENUITEM "Ș&tergere\tDel", ID_EDIT_DELETE
+        MENUITEM "&Redenumire", ID_EDIT_RENAME
         MENUITEM SEPARATOR
-        MENUITEM "&Copiază numele cheii", ID_EDIT_COPYKEYNAME
+        MENUITEM "&Copiere a numelui cheii", ID_EDIT_COPYKEYNAME
         MENUITEM SEPARATOR
         MENUITEM "&Găsire…\tCtrl+F", ID_EDIT_FIND
-        MENUITEM "Găsește &următorul\tF3", ID_EDIT_FINDNEXT
+        MENUITEM "Gă&sire următorul\tF3", ID_EDIT_FINDNEXT
     END
     POPUP "&Afișare"
     BEGIN
         MENUITEM "&Bara de stare", ID_VIEW_STATUSBAR
         MENUITEM SEPARATOR
-        MENUITEM "&Reglează separator", ID_VIEW_SPLIT
+        MENUITEM "&Reglare a separatorului", ID_VIEW_SPLIT
         MENUITEM SEPARATOR
-        MENUITEM "Împ&rospătează\tF5", ID_VIEW_REFRESH
+        MENUITEM "Î&mprospătare\tF5", ID_VIEW_REFRESH
     END
-    POPUP "Fa&vorite"
+    POPUP "&Favorite"
```

Comment:
Change requested: The &Favorite would collide with "&Fișier". Change is mandatory for that.

Reply:
The file is not localized in Romanian, so I cannot send you any proof. I didn't pay attention. I will revert this change.

Reply:
resolved